test/resources/Persons.spec.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var resources = require('../../lib/stripe').resources;
+var stripe = require('../../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+var ACCOUNT_TEST_ID = 'acct_123';
+var PERSON_TEST_ID = 'person_123';
+
+// Create new Person instance with pre-filled accountId:
+var person = new resources.Persons(
+  stripe,
+  {accountId: ACCOUNT_TEST_ID}
+);
+
+// Use spy from existing resource:
+person._request = stripe.customers._request;
+
+describe('Person Resource', function() {
+  describe('create', function() {
+    it('Sends the correct request', function() {
+      person.create({
+        first_name: "John",
+      });
```

Comment:
This part makes no sense to me. I know it works and why but that feels like a really weird way to approach this test and I'm ~confused why we even allow this.

Reply:
What part makes no sense?

Reply:
The part where to create a brand new object (which means said object does not exist yet) you use a pre-filled fake object.